.travis.yml

```diff
@@ -32,12 +27,51 @@ matrix:
         - docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 fiware/orion -dbhost mongodb
       before_script:
         - npm run lint -C ./validator
+       
+      script:
+         - node ./validator/validate.js -i common-schema.json -i geometry-schema.json -i specs/Weather/weather-schema.json -i specs/Alert/alert-schema.json -p specs -c true
+
+    - stage: test
+      name: "Unit Tests 3.6"
+      python: 3.6
+      node_js: 10
+
+      before_install:
+        - pip install flake8
+        # stop the build if there are Python syntax errors, PEP8 violations, undefined names
+        - flake8 . --count --select=E,F821,F822,F823 --max-line-length=127 --show-source --statistics
+        # exit-zero treats all errors as warnings.  GitHub editor is 127 chars wide
+        - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+      install:
+        ## Data model validator
+        - npm install -C ./validator
+        - sudo apt-get remove -y docker docker-engine docker.io
+        - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927
+        - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+        - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+        - sudo apt-get -qq update
+        - sudo apt-get install -y docker-ce
+        - docker run --name mongodb -d mongo:3.4
+        - docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 fiware/orion -dbhost mongodb
+      before_script:
+        - npm run lint -C ./validator
+       
+      script:
+         - node ./validator/validate.js -i common-schema.json -i geometry-schema.json -i specs/Weather/weather-schema.json -i specs/Alert/alert-schema.json -p specs -c true
+
+    - stage: test
```

Comment:
why do we need this?

Reply:
Previously the Documentation tests were subsumed into the `before_script` step of the validator tests. - i.e. `npm run lint -C ./validator` - however when this fails the CI *errors* rather than *fails* and the rest of the script does not run.

It is much easier to track down whether the JSON or Docs or harvesters are at fault if they are covered by separate test suites in separate test steps.

Reply:
Furthermore, because the linter has now been set up to *fail* on errors, Doc fails are more likely to be found sooner. - e.g. 1edda9f - There was a missing bracket on one of the new models.